ocatool/src/main.rs

```diff
@@ -20,18 +24,68 @@ struct Args {
 
 #[derive(Subcommand)]
 enum Commands {
+    /// Build oca objects out of ocafile
     Build {
         #[arg(short, long)]
         file: Option<String>,
+        #[arg(short, long)]
+        local_repository_path: Option<String>,
     },
+    /// Publish oca objects into online repository
     Publish {
         #[arg(short, long)]
         repository: String,
     },
+    /// Sign specific object to claim ownership
     Sign {
         #[arg(short, long)]
         scid: String,
     },
+    Show {
+        #[arg(short, long)]
+        said: String,
+        #[arg(short, long)]
+        local_repository_path: Option<String>,
+    },
+    Get {
+        #[arg(short, long)]
+        said: String,
+        #[arg(short, long)]
+        local_repository_path: Option<String>,
+    },
+    List {
+        #[arg(short, long)]
+        local_repository_path: Option<String>,
+    }
+
+}
+
+fn create_or_open_local_storage(path: PathBuf) -> SledDataStorage {
+    // TODO use PathBuf in SqlConfig to be more platform independent
+   let config = SledDataStorageConfig::build().path(path).unwrap();
+   let db = SledDataStorage::new()
+                .config(config);
+   return db;
+}
+
+
+fn get_oca_facade(local_repository_path: PathBuf) -> Facade {
+    let db = create_or_open_local_storage(local_repository_path);
```

Comment:
it would be better to nested dir for db files
```suggestion
    let db = create_or_open_local_storage(local_repository_path.join("oca_repository"));
```